### ISSUE TITLE
ScaladocParser swallows comments symbols within code blocks

### ISSUE THREAD

**Issue by Jentsch** [scalameta/scalameta#793]
While playing around with the upcoming Scaladoc parser I found a behaviour that I would classify as a minor bug.

Given I have Scaladoc where within a code block a comment is used like here:
```
/**
  * {{{
  *   // comment
  * }}}
  */
```
Then the ScaladocParser.parseScaladoc should return a Codeblock that includes the slashes (//).

I'm pretty sure the Bug is within the ScaladocParser and its interaction with [CommentOps](https://github.com/scalameta/scalameta/blob/master/scalameta/contrib/shared/src/main/scala/scala/meta/contrib/CommentOps.scala#L21) where the unconditional removal of all leading comment symbols seems to be problematic. But I'm not sure how to fix this.

Affected version: [current master](https://github.com/scalameta/scalameta/tree/93c9871363177d8a4ed5486d9b701d56adde0e57)

**Comment by olafurpg** [comment]
Thank you for reporting! @xavier-fernandez, can you take a look?

**Comment by Jentsch** [comment]
I may found a fix and created PR #794

**Comment by Jentsch** [comment]
Indention in code block seems to vanish, too.
```
/**
  * {{{
  * code (
  *   inner code
  * )
  * }}}
  */
```

produces a Codeblock with:
```
code (
inner code
)
```
instead of
```
code (
  inner code
)
```
